### ISSUE TITLE
Install data to install directory

### ISSUE THREAD

**Issue by paskino** [CCPPETMR/SIRF-SuperBuild#276]
The environment scripts now contain the variable `SIRF_INSTALL_PATH` which points to the installation directory. 

I suggest to use the directory `${SIRF_INSTALL_PATH}/share/sirf/data` to put the data in. 

This can also be used by the download [scripts ](https://github.com/CCPPETMR/SIRF-Exercises/blob/master/scripts/download_MR_data.sh#L81)of the `SIRF-Exercises` instead of the current `$SIRF_PATH` which points to the source directory (which resides in the build directory). 

Also this is compatible with conda installation where one can use `sys.prefix/share/sirf`

The same directory `$SIRF_INSTALL_PATH/share/ccpi` is used if `BUILD_CIL=ON` to store the data for CIL.

**Comment by KrisThielemans** [comment]
I don't think this is a SuperBuild issue, but SIRF. If you agree, please transfer the issue (bottom right corner).

Related is https://github.com/CCPPETMR/SIRF/issues/181 for SIRF installation, and https://github.com/CCPPETMR/CCPPETMR_VM/issues/116

I don't mind having `SIRF_INSTALL_PATH`,  could be quite non-intuitive what it means if people install it in `/usr/local` or indeed `/usr` once we rule the world. It sounds better to have a variable that would point `.../share/sirf/`, from where people could find the examples, the data, the doc. It will also allow us to install in `../share/sirf-x.x` at some point.

~bathomas and I were discussing location of data. In the future, you can imagine that people won't want to write in `../sirf/data` (or don't have permission for it). They would copy the data somewhere else play around with it (relevant on Azure and the VM as we're afraid of running out of disk-space). But that then implies that we'd need a `SIRF_DATA_PATH` environment variable...

Not sure if we can/should do this now though. At the moment, we can work-around it with symlinks (horrible, I know).
